ui/src/app/cohort-search/tree/tree.component.tsx

```diff
@@ -188,6 +188,15 @@ export const CriteriaTree = withCurrentWorkspace()(class extends React.Component
       && domainId !== Domain.VISIT.toString();
   }
 
+  get treeContainerStyle() {
+    const {node: {domainId}} = this.props;
+    const treeContainerStyle = {paddingTop: '3rem', width: '99%'};
+    if (!this.showHeader) {
+      treeContainerStyle.paddingTop = domainId === Domain.VISIT.toString() ? '0.5rem' : '2.5rem';
```

Comment:
can we add paddingBottom to searchBarContainer instead to avoid 2 checks for Domain Visit here and to show searchBar

Reply:
We needed it here because the searchBarContainer had `position: absolute` set and adding padding to the bottom wouldn't have made a difference. But I just realized since we recently implemented sticky headers and now scroll within the tree, we don't need the search bar to have absolute positioning anymore and we can also reduce the conditions on paddingTop to one. Makes things a lot simpler :)